proxy/ProxyProtocol.cc

```diff
@@ -0,0 +1,149 @@
+/** @file
```

Comment:
Do you mean to put ProxyProtocol.cc in the tree twice?  Both in proxy and iocore/net.

Reply:
I had moved the ProxyProtocol.cc from proxy/ into iocore/net, then rebased and sqquash my commits, but somehow it is still in both places.  I will fix it now.